docs-src/go/how-to-send-a-query-to-a-workflow-execution-in-go.md

```diff
@@ -30,18 +30,13 @@ if err != nil {
 // ...
 ```
 
-The `QueryWorkflowWithOptions()` API provides similar functionality, but with the ability to set additional configurations through [QueryWorkflowWithOptionsRequest](https://pkg.go.dev/go.temporal.io/sdk/client#QueryWorkflowWithOptionsRequest).
-When using this API, you will also receive a structured response of type [QueryWorkflowWithOptionsResponse](https://pkg.go.dev/go.temporal.io/sdk/client#QueryWorkflowWithOptionsResponse).
+The `response` returned by the Query needs to be decoded into a `result`.
```

Comment:
```suggestion
The value of `response` returned by the Query needs to be decoded into `result`.
```
Per https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements.